include/tcpdf_asn1.min.php

```diff
@@ -0,0 +1,116 @@
+<?php
+// ASN.1 Parser start 21:31 Sore Kamis 26 Maret 2009
+// ASN.1 Parser at 22:10 Sore Kamis 26 Maret 2009 Telah jadi utk standar asn.1
+// 
+// 06:40 Esuk Jumat 27 Maret 2009 ASN.1 Parser kesulitan dlm memecahkan explicit > 9
+
+// 11:18 Esuk Jumat 27 Maret 2009 parse explicit:xx mulai dipecahkan. kemungkinan tlh jadi
+// 17:51 Sore Jumat 27 Maret 2009 memecahkan explicit sampai 2097151 (65536 * 32) kurang 1
+
+// 20:04 Sore Jumat 27 Maret 2009 ASN.1 Parser tlh jadi. Congratulation....
+// 12:15 Sore 16/05/2023
```

Comment:
all this file seems quite old, what is it source ?
what is the license of this code ?

Reply:
no, it useless. just progress mark. i'm forget to remove some comment line.